### PR TITLE
Updated Editor Static Rigid Component card

### PR DESCRIPTION
**Pull request by moraaar**
image

The text also wraps correctly. Thanks to @cgalvan for adding support for it!

image

Signed-off-by: moraaar <moraaar@amazon.com>

### CODE REVIEW

File: Code/Framework/AzFramework/AzFramework/Physics/NameConstants.cpp [hunk 1]
```diff
@@ -45,6 +45,12 @@ namespace Physics
             return interpunct;
         }
 
+        const AZStd::string& GetBulletPoint()
+        {
+            static const AZStd::string interpunct = "\xE2\x80\xA2";
```
[the remark this answers is not in the document]
Comment (moraaar) [reply Code/Framework/AzFramework/AzFramework/Physics/NameConstants.cpp:50]: Thank! Copy paste from the previous function :)

**Review by SergeyAMZN — APPROVED** [review]

**Review by hultonha — APPROVED** [review]
Nice! Tagging @bhanuja-s to take a look too.

**Review by bhanuja-s — APPROVED** [review]
This looks great, thank you @moraaar!

**Comment by cgalvan** [comment]
Woo! :)

**Pull request merged**